cmd/shfmt/shfmt.1.scd

```diff
@@ -131,7 +131,10 @@ keep_padding       = true
 # --func-next-line
 function_next_line = true
 
-# Ignore the entire "third_party" directory.
+# Ignore the entire "third_party" directory when formatting directories,
+# such as "shfmt -l -w .".
```

Comment:
```suggestion
# Ignore the entire "third_party" directory when operating on a directory,
# e.g. "shfmt -l -w ." or "shfmt -f some/dir".
```

Reply:
not sure I follow - is the addition of a second example what makes it clearer? both examples are pretty much the same, what matters is that the argument is a directory. I'd prefer to keep the man page short in general.

Reply:
> not sure I follow - is the addition of a second example what makes it clearer? both examples are pretty much the same, what matters is that the argument is a directory.

And that exactly needs to be pointed out more precisely. It ignores *the entire "third_party" directory*, **if and only if** a directory is passed to `shfmt`. It does nothing otherwise. It is not only working "when formatting" directories, but whenever the user operates on a folder. And this statement will be ignored if a file is passed directly. 

> I'd prefer to keep the man page short in general.

Me, too. 

Maybe something like this then?

```suggestion
# Ignore the entire "third_party" directory when calling shfmt on directories, e.g. "shfmt -l -w .". 
```

Reply:
Thanks, done.